### ISSUE TITLE
Trezor T - Could not open client ... image not found

### ISSUE THREAD

**Issue by Sjors** [achow101/HWI#38]
If I don't insert or unlock the device enumerate returns `[]` as expected. Once unlocked I get:

```
./hwi.py enumerate
INFO:trezorlib.transport:looking for device by full path: hid:IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/XHC1@14/XHC1@14000000/HS10@14600000/TREZOR@14600000/TREZOR Interface@1/IOUSBHostHIDDevice@14600000,1
[{"type": "trezor", "path": "IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/XHC1@14/XHC1@14000000/HS10@14600000/TREZOR@14600000/TREZOR Interface@1/IOUSBHostHIDDevice@14600000,1", "serial_number": "#################", "error": "Could not open client or get fingerprint information: dlopen(libusb-1.0.dylib, 6): image not found"}]

# serial number censored by me
```

**Comment by achow101** [comment]
I don't have a Trezor T so I can't test this. You should check the https://github.com/trezor/python-trezor if there is anything specific about it that is necessary.

**Comment by matejcik** [comment]
Trezor T communicates over webusb (implemented by libusb), not hidapi. I'm honestly not sure what's going on in your error message - the command is complaining that `libusb-1.0.dylib` is not available, which means that your libusb isn't installed properly.
But I suspect that this would just fail a little bit later because HWI forces a path that starts with "hid:". Trezor T exposes a HID interface for U2F but you won't be able to send commands to it.

**Comment by yashbhutwala** [comment]
Hey guys, I'm getting the same error as @Sjors but with a Trezor One.  I'm not sure if it's something to do with libusb1 dependency on Mac?

**Comment by matejcik** [comment]
@yashbhutwala are you by any chance on firmware 1.7.1 ? that uses webusb too.

**Comment by yashbhutwala** [comment]
@matejcik I'm on firmware version 1.6.3.  Does that also use webusb for Trezor One?

**Comment by matejcik** [comment]
no, 1.6.3 uses hid. that means your problem is caused by something else

pá 9. 11. 2018 14:30 odesílatel Yash <notifications@github.com> napsal:

**Comment by yashbhutwala** [comment]
@matejcik Hmm, ok the error I'm getting is about the U2F interface it seems and I don't really need that...

`
"error": "Could not open client or get fingerprint information: hid device not found: hid:IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/XHC1@14/XHC1@14000000/HS02@14200000/TREZOR@14200000/U2F Interface@1/IOUSBHostHIDDevice@14200000,1"
`
